### PR TITLE
PbA: Add Powered by Aragon uses cases & action link

### PR DESCRIPTION
**Pull request by Evalir**
Adds the uses cases for "Powered by Aragon". Opening as a draft until issues are resolved:

- [ ] Correct copy for CTA Link
- [ ] Link to new website

### CODE REVIEW

File: src/components/Home/Powered.js [hunk 1]
```diff
@@ -62,6 +62,20 @@ const Powered = props => (
...
     </CardsContainer>
+    <div>
+      <a
+        className="action-button powered-link centered"
+        href="https://poweredbyaragon.com"
+      >
+        <span>
+          <FormattedMessage
+            id="home.organizations.action"
+            defaultMessage="Learn more"
```
Comment (Evalir) [inline src/components/Home/Powered.js]: Once we have the correct copy for this link, we should update the i18n for this.

File: src/components/Home/Powered.js [hunk 2]
```diff
@@ -62,6 +62,20 @@ const Powered = props => (
         </div>
       </Card>
     </CardsContainer>
+    <div>
+      <a
+        className="action-button powered-link centered"
+        href="https://poweredbyaragon.com"
```
Comment (Evalir) [inline src/components/Home/Powered.js]: Before merging we need to update this link.

**Review by Evalir — COMMENTED** [review]
Leaving the issues also as comments, so we can discuss. cc @ganeJackS

**Comment by sohkai** [comment]
@Evalir should we close this PR?

**Comment by Evalir** [comment]
Yes; let's close this.

**Pull request closed without merging**